app/Http/Controllers/CustomizerController.php

```diff
@@ -250,6 +250,11 @@ protected function prepSeed(Request $request, bool $save = false)
         $rand = new Randomizer([$world]);
 
         $rand->randomize();
+
+        foreach ($request->input('texts', []) as $key => $value) {
+            $world->setText($key, $value);
```

Comment:
This could be problematic if the key doesn't exist? Should we verify that it is a proper key?